aries_cloudagent/wallet/routes.py

```diff
@@ -103,6 +108,30 @@ class DIDEndpointWithTypeSchema(OpenAPISchema):
     )
 
 
+class JWSCreateSchema(OpenAPISchema):
+    """Request schema to create a jws with a particular DID."""
+
+    headers = fields.Dict()
+    payload = fields.Dict(required=True)
+    did = fields.Str(description="DID of interest", required=True, **INDY_DID)
+    verification_method = fields.Str(
+        data_key="verificationMethod",
+        required=True,
+        description="Information used for proof verification",
+        example=(
+            "did:key:z6Mkgg342Ycpuk263R9d8Aq6MUaxPn1DDeHyGo38EefXmgDL"
+            "#z6Mkgg342Ycpuk263R9d8Aq6MUaxPn1DDeHyGo38EefXmgDL"
+        ),
+        validate=Uri(),
+    )
```

Comment:
I think this should be an either or thing. If the DID is given, we'll select the first appropriate key from the DID Doc and use that for the kid (in practice, we assume the appropriate key is `#keys-1`). If the verification method is given, we use that as the kid directly. Given this, I think we want `required=False` on both fields and then we'll perform more checks in the handler to make sure at least one is present.

Reply:
good catch!